pkg/balancer-js/src/utils/errors.ts

```diff
@@ -157,6 +157,7 @@ const balancerErrorCodes: Record<string, string> = {
   '601': 'FLASH_LOAN_FEE_PERCENTAGE_TOO_HIGH',
   '602': 'INSUFFICIENT_FLASH_LOAN_FEE_AMOUNT',
   '603': 'AUM_FEE_PERCENTAGE_TOO_HIGH',
+  '998': 'UNIMPLEMENTED',
```

Comment:
We have an "INVALID_OPERATION". Maybe "UNSUPPORTED_OPERATION"?

Reply:
Eh I'd say this is different, there's no 'operation'. It's just a piece from an abstract contract we choose to not implement.